cypress/support/pages/admin/pageAdministration.js

```diff
@@ -269,5 +277,68 @@ class Management {
       .find('input')
       .click({ force: true })
   }
+
+  schoolNumberIsAdded() {
+    cy.get(Management.#schoolNumberForm)
+        .should('not.be.empty')
+  }
+
+
+  migrationButtonIsEnabled() {
+    cy.get(Management.#startMigrationButton)
+        .should('be.enabled')
+  }
+
+  clickStartMigration() {
+    cy.get(Management.#startMigrationButton)
+        .click()
+  }
+
+  migrationInformationIsVisible() {
+    cy.get(Management.#migrationInformationText)
+        .should('be.visible')
+  }
+
+  clickSupportLink() {
+    cy.contains("a", "Support")
```

Comment:
its better to use some `data-testid` here.

Reply:
Thank you for the input, it's corrected.